inductiva/utils/files.py

```diff
@@ -183,10 +190,11 @@ def download_from_url(url: str, unzip: bool = False) -> str:
 
     # Unzip all files as they were zipped.
     if unzip and zipfile.is_zipfile(downloaded_to):
-        resulting_path = pathlib.Path(downloaded_to).with_suffix("")
+        resulting_path = (pathlib.Path(path)
+                          if path else downloaded_to.with_suffix(""))
```

Comment:
Should it be `if path else pathlib.Path(downloaded_to).with_suffix("")`?

Reply:
Actually, no, because the variable ```downloaded_to``` is already of type ```<class 'pathlib.PosixPath'>```. I printed the types to verify, with ```path=None```:

```python
# ...
    resulting_path = downloaded_to
    print(type(resulting_path))
    # Unzip all files as they were zipped.
    if unzip and zipfile.is_zipfile(downloaded_to):
        resulting_path = (pathlib.Path(path)
                          if path else downloaded_to.with_suffix(""))
        print(type(resulting_path))
# ...
```